worker-taskstowing/src/main/java/com/microfocus/caf/worker/taskstowing/TaskStowingBulkWorker.java

```diff
@@ -0,0 +1,231 @@
+                taskApiVersionList.add(workerTask.getVersion());
+                taskDataList.add(workerTask.getData());
+                taskStatusList.add(workerTask.getStatus().name());
+                contextList.add(contextBytes);
+                toList.add(workerTask.getTo());
+                trackingInfoList.add(trackingInfoBytes);
+                sourceInfoList.add(sourceInfoBytes);
+                correlationIdList.add(workerTask.getCorrelationId());
+
+                validatedWorkerTasks.add(workerTask);
+            } catch (final JsonProcessingException jsonProcessingException) {
+                LOGGER.error(TaskStowingWorkerFailure.FAILED_TO_SERIALIZE_TASK, jsonProcessingException);
+                workerTask.setResponse(createFailureResult(TaskStowingWorkerFailure.FAILED_TO_SERIALIZE_TASK, jsonProcessingException));
+            }
+        }
+
+        if (validatedWorkerTasks.isEmpty()) {
+            // An error will already have been logged and a response set on each task that failed validation, so just return.
+            return;
+        }
+
+        if (anyListsEmptyOrNotSameSize(validatedWorkerTasks,
+                                       partitionIdList,
+                                       jobIdList,
+                                       taskClassifierList,
+                                       taskApiVersionList,
+                                       taskDataList,
+                                       taskStatusList,
+                                       contextList,
+                                       toList,
+                                       trackingInfoList,
+                                       sourceInfoList,
+                                       correlationIdList)) {
+            // Should not happen normally, but want to make sure we don't go ahead with the database insertion if it does.
+            LOGGER.error(TaskStowingWorkerFailure.UNKNOWN_ERROR);
+            validatedWorkerTasks.forEach(workerTask -> {
+                workerTask.setResponse(createFailureResult(TaskStowingWorkerFailure.UNKNOWN_ERROR));
+            });
+            return;
+        }
+
+        LOGGER.info("{}/{} task(s) were validated without errors, will now attempt to stow {} task(s)",
+                    validatedWorkerTasks.size(), workerTasks.size(), validatedWorkerTasks.size());
+        try {
+            databaseClient.insertStowedTasks(
+                partitionIdList,
+                jobIdList,
+                taskClassifierList,
+                taskApiVersionList,
+                taskDataList,
+                taskStatusList,
+                contextList,
+                toList,
+                trackingInfoList,
+                sourceInfoList,
+                correlationIdList);
+
+            validatedWorkerTasks.forEach(workerTask -> {
+                workerTask.setResponse(createSuccessResultNoOutputToQueue());
+            });
+        } catch (final Exception exception) {
```

Comment:
Again we're missing transient issue detection here.  Note that if we do detect a transient issue we need to call the `workerTask.setResponse()` overload that takes a `TaskRejectedException`, and we need to call it in a loop just like the success and hard error cases.